### PR TITLE
NIAD-145: Fixing handlers import after moving them from mhs_common to common

### PR DESCRIPTION
**Pull request by bartek-sarul**

### CODE REVIEW

File: nhais/main.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import tornado.httpclient
 import utilities.integration_adaptors_logger as log
-from requests import healthcheck_handler
+from mhs_common.request import healthcheck_handler
```
Comment (grzegorz-robak) [inline nhais/main.py]: This should be `mhs_common.handlers`, shouldn't it?

**Review by grzegorz-robak — CHANGES_REQUESTED** [review]

File: nhais/main.py (hunk 1 above)
Comment (bartek-sarul) [reply nhais/main.py]: no, it's in "common" now as it's shared with NHAIS

Comment (bartek-sarul) [reply nhais/main.py]: I agree, that there is still something wrong with this approach as we don't uniquely identify sub-dependency packages but let's merge this and then work on better solution

Comment (bartek-sarul) [reply nhais/main.py]: ahh damn, yes you are right

**Review by grzegorz-robak — APPROVED** [review]

**Review by mattd-kainos — APPROVED** [review]

**Review by Alicja-Kainos — APPROVED** [review]

**Pull request merged**